example/example/tests/test_customer_views.py

```diff
@@ -35,6 +36,22 @@ def test_create_customer(self, user, mock_storage):
         client.credentials(HTTP_AUTHORIZATION=f"Token {key}")
         url = reverse("customers-list")
 
+        # a user w/out pending_customer cannot create a customer...
+
+        assert user.pending_customer is False
+
+        response = client.post(url, customer_data, format="json")
+        content = response.json()
+
+        assert status.is_client_error(response.status_code)
+        assert Customer.objects.count() == 0
+        assert content["detail"] == HasPendingCustomer.message
+
+        # a user w/ pending_customer can create a customer...
+
+        user.pending_customer = True
+        user.save()
```

Comment:
This test seems to do one thing i.e. post customer data to prove the user is not created, but just do `user.save`, to prove it can. I would expect these to be 2 separate tests, or at the very least to both be post requests, to verify the API works consistently.

Reply:
They could be separate tests:  one to check that a user w/ `pending_customer` set to `False` _cannot_ create a `customer` and one to check that a user w/ `pending_customer` set to `True` _can_ create a `customer`.

I will do that.

Reply:
Done in 12c396f3af144df369b9292f23b70024fc83e707